util/password.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+  "crypto/rand"
+  "golang.org/x/crypto/bcrypt"
+)
+
+
+const passwordLength = 16
+const hashCost = 12
+
+func RandomPassword() string{
```

Comment:
Yes. Because first we generate a random password, then mail it to the company(for this purpose we need the un-encrypted password) and then encrypt and save it.